assets/model_monitoring/components/tests/unit/test_numerical_data_drift_metrics.py

```diff
@@ -1,87 +1,413 @@
+     "x_mean": 50,
+     "y_mean": 53,
+     "x_std_dev": 15,
+     "y_std_dev": 5,
+     "x_obs": 100_000,
+     "y_obs": 10_000,
+    },
+    {
+     "name": "3a",
+     "scenario": "Large drift. Bound distnace measures (Jensen-Shannon) approaching maximum theoretical values.",
+     "x_mean": 50,
+     "y_mean": 100,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 100_000,
+     "y_obs": 100_000,
+    },
+    {
+     "name": "3b",
+     "scenario": "Extreme drift scenario. Bound distnace measures (JSD) should reach their maximum theoretical value.",
+     "x_mean": 50,
+     "y_mean": 250,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 100_000,
+     "y_obs": 100_000,
+    },
+    {
+     "name": "4a",
+     "scenario": "Extreme drift scenario. All values the same, but different in baseline and prod.",
+     "x_mean": 50,
+     "y_mean": 51,
+     "x_std_dev": 0,
+     "y_std_dev": 0,
+     "x_obs": 9_000,
+     "y_obs": 9_000,
+    },
+    {
+     "name": "5a",
+     "scenario": "Distance between identical distributions should be exaclty 0.",
+     "x_mean": 50,
+     "y_mean": 50,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 9_000,
+     "y_obs": 9_000,
+    },
+    {
+     "name": "5b",
+     "scenario": "Distance between two samples of the different sizes of the same distributions should be close to 0.",
+     "x_mean": 50,
+     "y_mean": 50,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 9_000,
+     "y_obs": 900,
+    },
+    {
+     "name": "5c",
+     "scenario": "Non-zero distance between distributions with the same mean and sample size, but differnt std_dev.",
+     "x_mean": 50,
+     "y_mean": 50,
+     "x_std_dev": 15,
+     "y_std_dev": 10,
+     "x_obs": 9_000,
+     "y_obs": 9_000,
+    },
+    {
+     "name": "6a",
+     "scenario": "Confirming distance of moderately drifted distribution against Scipy.",
+     "x_mean": 50,
+     "y_mean": 51,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 9_000,
+     "y_obs": 9_000,
+    },
+    {
+     "name": "6b",
+     "scenario": (
+         "Weak distance value dependence on sample size. "
+         "Distance should be close to test 6a (might not be the case for KS-Test)."),
+     "x_mean": 50,
+     "y_mean": 51,
+     "x_std_dev": 15,
+     "y_std_dev": 15,
+     "x_obs": 9_000,
+     "y_obs": 900,
+    },
+    {
+     "name": "6c",
+     "scenario": "Decreasing std_dev in prodcution dataset. Increase in distance expected when compared to test 6a.",
+     "x_mean": 50,
+     "y_mean": 51,
+     "x_std_dev": 15,
+     "y_std_dev": 5,
+     "x_obs": 9_000,
+     "y_obs": 9_000,
+    },
+    {
+     "name": "6d",
+     "scenario": "Decreasing both sample size and std_dev in production dataset. Similar values as test 6c expected.",
+     "x_mean": 50,
+     "y_mean": 51,
+     "x_std_dev": 15,
+     "y_std_dev": 5,
+     "x_obs": 9_000,
+     "y_obs": 900,
+    },
```

Comment:
One question for curiosity, what is the difference between scenario group 2(2a to 2d) vs group 6(6a to 6d)? From what I am reading, they are pretty similar.

Reply:
Yes they are very similar. The reason why I am including both is because the current implementation of the KS test is capped at 10k samples. I wanted to include tests that covered about 100k samples, as well as some in the upper limit of what KS can handle.

Reply:
Got it. Thanks for answering.